lib/logstash/inputs/heartbeat.rb

```diff
@@ -3,6 +3,7 @@
 require "logstash/namespace"
 require "stud/interval"
 require "socket" # for Socket.gethostname
+require "logstash/plugin_mixins/ecs_compatibility_support"
```

Comment:
deprecation logger support should be required...

```suggestion
require "logstash/plugin_mixins/deprecation_logger_support"
require "logstash/plugin_mixins/ecs_compatibility_support"
```